cmd/state/internal/cmdtree/projects.go

```diff
@@ -6,10 +6,11 @@ import (
 	"github.com/ActiveState/cli/internal/output"
 	"github.com/ActiveState/cli/internal/runners/projects"
 	"github.com/ActiveState/cli/pkg/platform/authentication"
+	"github.com/spf13/viper"
 )
 
 func newProjectsCommand(outputer output.Outputer, auth *authentication.Auth) *captain.Command {
-	runner := projects.NewProjects(outputer, auth)
+	runner := projects.NewProjects(outputer, auth, viper.GetViper())
```

Comment:
Fyi filed this story to address - https://www.pivotaltracker.com/story/show/173734115